Fixes build by not treating all warnings as errors, no packaging system
should ever do this unless you are in a development environment.

diff --git a/Makefile b/Makefile
index dc0afb9..5c63e0d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 PREFIX?=/usr/local
 BINDIR=$(DESTDIR)$(PREFIX)/bin
 
-override CFLAGS+=-Wall -Werror -Wformat=2 -g
+override CFLAGS+=-Wall -Wformat=2 -g
 override LDFLAGS?=
 
 LDLIBS?=
